### DISCUSSION TITLE
Build error

### DISCUSSION THREAD

**Discussion by ednsinf** [unoplatform/uno#18332] [Q&A]
Stupid framework not build after install all dependencies

1>C:\Users\xxx\.nuget\packages\uno.wasm.bootstrap\8.0.21\build\Uno.Wasm.Bootstrap.targets(226,3): error :    at Uno.Wasm.Bootstrap.ShellTask_v038af955a9b67691b2e0a0ec4ebf87820919b6b1.PrepareFinalDist() in /_/src/Uno.Wasm.Bootstrap/ShellTask.cs:line 1632
1>C:\Users\xxx\.nuget\packages\uno.wasm.bootstrap\8.0.21\build\Uno.Wasm.Bootstrap.targets(226,3): error :    --- End of inner exception stack trace ---
1>C:\Users\xxx\.nuget\packages\uno.wasm.bootstrap\8.0.21\build\Uno.Wasm.Bootstrap.targets(226,3): error :    at Uno.Wasm.Bootstrap.ShellTask_v038af955a9b67691b2e0a0ec4ebf87820919b6b1.PrepareFinalDist() in /_/src/Uno.Wasm.Bootstrap/ShellTask.cs:line 1637
1>C:\Users\xxx\.nuget\packages\uno.wasm.bootstrap\8.0.21\build\Uno.Wasm.Bootstrap.targets(226,3): error :    at Uno.Wasm.Bootstrap.ShellTask_v038af955a9b67691b2e0a0ec4ebf87820919b6b1.Execute() in /_/src/Uno.Wasm.Bootstrap/ShellTask.cs:line 277
1>Done building project "MyTestProject.WebApp.csproj" -- FAILED.
1>MyTestProject.WebApp -> C:\Projetos\MyTestProject.WebApp\MyTestProject.WebApp\bin\Debug\net8.0-android\MyTestProject.WebApp.dll

**Comment by jeromelaban** [comment]
Thanks for the report.

**Please make sure to follow the [code of conduct of this repository](https://github.com/unoplatform/uno/blob/master/CODE_OF_CONDUCT.md). This kind of language is not accepted here.**

If you can provide a [more detailed log](https://platform.uno/docs/articles/uno-builds-troubleshooting.html), it will be useful.
